xm-commons-domain-event/src/main/java/com/icthh/xm/commons/domain/event/domain/HttpDomainEventPayload.java

```diff
@@ -0,0 +1,22 @@
+package com.icthh.xm.commons.domain.event.domain;
```

Comment:
please rename all packages from 
`com.icthh.xm.commons.domain.event.*` to `com.icthh.xm.commons.domainevent.*`